docs/BasicCommunication/GetSystemTime/index.md

```diff
@@ -0,0 +1,109 @@
+## GeSystemTime
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Obtain current UTC time  
+
+When an application needs to start a protected service with SDL core the app sends a certificate to SDL during the TLS or DTLS handshake process.  
+In order to validate the certificate and to check whether this certificate is not expired SDL core needs an accurate UTC system time value.
+
+### Request
+SDL sends a GetSystemTime request after receiving a BC.OnSystemTimeReady notification from the HMI.  
+After sending the request SDL starts [`<DefaultTimeout>`](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Configuration%20file/index.md#main) (value from .ini file) waiting for response from HMI. 
+
+#### Parameters
+
+### Response
+
+!!! must  
```

Comment:
Need closing `!!!`

Reply:
@jacobkeeler, thank you
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/d5cb22355a29c59cb210659f179ed201a455b5b3